WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -204,6 +203,9 @@ public static async Task<ExitCode> RunAsGuiAsync(this WasabiApplication app)
 				else
 				{
 					appBuilder.StartWithClassicDesktopLifetime(app.AppConfig.Arguments);
+
+					MainViewModel.Instance.ClearStacks();
```

Comment:
In normal exit (shutdown) we can use Exit event from desktop lifetime:
```
				desktop.Exit += (sender, args) =>
				{
					MainViewModel.Instance.ClearStacks();
					MainViewModel.Instance.StatusIcon.Dispose();
				};
```

and put it here:
https://github.com/zkSNACKs/WalletWasabi/blob/33233a26c839aa06771847c6dd8099e61f7df45b/WalletWasabi.Fluent/App.axaml.cs#L53C35-L53C35

As for Avalonia inpact in general `MainViewModel.Instance.StatusIcon.Dispose();` and `MainViewModel.Instance.ClearStacks();` should not have any impact, but looking at `MainViewModel.Instance.ClearStacks()`; code we might do some code that can trigger some UI events, not sure we can test it right now at full extent (test all possible navigation states).

Reply:
> It's the best what we can do right now imho.

Agree